github_copilot_seats/seats.py

```diff
@@ -0,0 +1,113 @@
+import json
+import os
+from datetime import datetime, timezone, timedelta
+from typing import Any
+import autokitteh
+from autokitteh.github import github_client
+from autokitteh.slack import slack_client
+from helpers import github_username_to_slack_user_id
+
+GITHUB_ORG = os.getenv("GITHUB_ORG")
+IDLE_USAGE_THRESHOLD = int(os.getenv("IDLE_USAGE_THRESHOLD"))
+LOGINS = os.getenv("LOGINS")
+LOG_CHANNEL = os.getenv("LOG_CHANNEL")
+
+logins = LOGINS.split(",") if LOGINS else None
```

Comment:
1. Why not put this condition directly in the assignment to `LOGINS` in line 12?
2. Why `else None`? If you remove it, and the env var is empty, you get an empty list - which doesn't cause loops to crash, and also evaluates to `False` if checked as a Boolean

Reply:
Do you mean like this?
```
LOGINS = os.getenv("LOGINS").split(",")
```
That doesn't work, because instead of an empty list, I get `['']`

Reply:
You're right! In that case, add `else []`, but not `else None` - so at least `logins` will always have the same type.

It doesn't actually help to simplify line 39, but it's still safer.